src/modules/tas_studio/editor/mod.rs

```diff
@@ -296,6 +305,23 @@ struct InsertCameraLineAdjustment {
     did_split: bool,
 }
 
+/// Camera view edit modes.
+///
+/// `Alt` adjusts differently for different kinds of line.
```

Comment:
Could you describe here what exactly it does?

Reply:
Alt is in case you want the result to be something like target yaw offset or target yaw velocity lock.